pysqldb3/pysqldb3.py

```diff
@@ -2136,19 +2136,17 @@ def table_to_gpkg(self, table, gpkg_name, gpkg_tbl = None, schema=None, path=Non
                                  path=path, gpkg_name=gpkg_name, cmd=cmd, gdal_data_loc=gdal_data_loc, gpkg_tbl = gpkg_tbl,
                                  print_cmd=print_cmd, srid=srid)
     
-    def gpkg_to_table(self, dbo, gpkg_name, gpkg_tbl, path=None, schema=None, table =None, 
+    def gpkg_to_table(self, gpkg_name, gpkg_tbl, path=None, schema=None, table =None, 
                      srid=2263, port=None, gdal_data_loc=GDAL_DATA_LOC, precision=False, private=False, temp=True,
                      gpkg_encoding=None, print_cmd=False, days=7, bulk_upload = False):
         """
         Imports single geopackage table to database. This uses GDAL to generate the table.
-        :param dbo: Database connection
-        :param path: File path of the shapefile
-        :param schema: Schema to use in the database (defaults to db's default schema)
         :param gpkg_name: Geopackage name (ends in .gpkg)
         :param gpkg_tbl: Input table name from Geopackage.
+        :param path: File path of the shapefile
```

Comment:
should be path to geopackage, unless thats in the gpkg_name in which case this can be removed